GREEDY/DataManagers/CategoryManager.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using GREEDY.Models;
+using GREEDY.Data;
+using System.Linq;
+
+namespace GREEDY.DataManagers
+{
+    public class CategoryManager : ICategoryManager
+    {
+        public List<string> GetAllDistinctCategories()
+        {
+            using (DataBaseModel context = new DataBaseModel())
+            {
+                var distinctCategories = context.Set<CategoryDataModel>()
+                    .Select(x => x.Category).Distinct();
+                return distinctCategories.ToList();
+            }
+        }
+        /*public int AddItems(IEnumerable<Item> items, Shop shop, string username)
```

Comment:
any comments there? why 2/3 class is commented? is it not working? or do you need time to fix it?

Reply:
Woops will fix this, this code shouldn't even be here